pages/cart/cart.css

```diff
@@ -0,0 +1,66 @@
+@import url(../../basic.css);
+main {
+    margin-top: 2rem;
+}
+
+.cart-container {
+    display: flex;
+    flex-direction: row;
```

Comment:
use this in the class directly from the basic.css you have defined instead of using it here